public/data/heroes.json

```diff
@@ -0,0 +1,7558 @@
+{
```

Comment:
We have a `lib/characters` file which would probably need refactoring to include the charms stuff 😅 but i assume this is C+P from your own project